controllers/RatingController.js

```diff
@@ -30,7 +32,12 @@ export default class RatingController {
       if (article === null) {
         return notFoundError(req, res);
```

Comment:
Hey, is there any reason why you're not sending back an error message that the article wasn't found?